### PR TITLE
Tribute Page: added the tech stack requirement and eliminated all ESLint warnings.

### PR DESCRIPTION
**Pull request by tchaffee**
#### Pre-Submission Checklist

- [x] Your pull request targets the `master` branch of freeCodeCamp/testable-projects-fcc
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] Your changes have been tested either locally or using a newly created CDN based on your fork's testable-projects-fcc/build/bundle.js file
- [x] Working changes have been added to the build by running `npm run build`

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.

- [ ] Closes currently open issue: Closes #XXX

#### Description

Added the tech stack requirement and eliminated all ESLint warnings. The hope is that these are the final changes to this file for this phase of the project.

### CODE REVIEW

**Comment by tchaffee** [comment]
@tbushman This has now been rebased and no longer has conflicts. Could you please review when you get a chance?

File: src/project-tests/tribute-page-tests.js [hunk 1]
```diff
@@ -9,154 +10,185 @@ export default function createTributePageTests() {
...
-      it(`1. My tribute page should have an element with corresponding
-        id="main", which contains all other elements.`, function() {
-          FCC_Global.assert.isNotNull(document.getElementById('main'));
-          FCC_Global.assert(
-            document.querySelectorAll('#main div, #main a, #main h1, #main img')
-            .length,
-            'element with id="main" must contain other elements');
+      let reqNum = 0;
+
+      reqNum++;
```
Comment (tbushman) [inline src/project-tests/tribute-page-tests.js]: Sorry to bug on this, but were you going to do a mocha beforeEach() for incrementing reqNum ?

Comment (tchaffee) [inline src/project-tests/tribute-page-tests.js]: I was. And it didn't work! So I reverted to this method. Thanks for asking.

**Comment by tbushman** [comment]
LGTM

**Pull request merged**